### PR TITLE
Showing team description when hovering over a team

### PR DESCRIPTION
**Pull request by B1s9l**
Issue #99

# Pull Requests Checklist

Assignee:

- \[ \] code commented?
- \[ \] deactivated things not needed? (e.g. other features which only were prepared)
  - \[ \] alerts that this is todo?
- \[ \] test run localhost?
- \[ \] tested manually all features and edge cases?
- \[ \] documentation added / updated?
- \[ \] pre-commit checks passed?
- \[ \] pipelines passed?

Reviewer:

- \[x] correct branch?
- \[x] are there any code smells?
- \[x] did pipeline pass?
- \[ \] is the code well documented?
- \[x] tested manually all features and edge cases?

### CODE REVIEW

File: src/components/views/TeamsOverview.tsx [hunk 1]
```diff
@@ -57,18 +53,19 @@ const TeamsOverview = () => {
...
   const goTeam = (teamId) => {
-    navigate(`/teams/${teamId}`); //change routing to point at created team
+    navigate(`/teams/${teamId}`);
   };
 
   return (
     <BaseContainer>
-      <h1 className="teams-overview header">{userName}&#39;s Teams</h1>
+      <h1 className="teams-overview header">{userName}s Teams</h1>
       <div className="teams-overview container">
```
Comment (Monti934) [inline src/components/views/TeamsOverview.tsx:62]: please add Apostroph again

**Review by Monti934 — CHANGES_REQUESTED** [review]

**Review by Monti934 — APPROVED** [review]
thanks for change
